tests/sc/README.md

```diff
@@ -1,6 +1,7 @@
 Testy extended_* poszerzają istniejące test próbując wyłapać dodatkowe przypadki brzegowe.
 Test extended_pipe_closed uruchamiany dla więcej niż 6 programów wykracza poza wymaganą specyfikację zadania.
 Testy send_recv_1.self oraz reasonable_sized_data.self sprawdzają w bezpośredni sposób przesyłanie dużych wiadomości.
+Testy big_brodcast.self oraz reasonable_sized_brodcast.self sprawdzają przesyłanie dużych wiadomości przez MIMPI_REDUCE.
```

Comment:
Zdaje się, że to powinny być dwie oddzielne linijki: jedna o `Bcast`, druga o `Reduce`.